La-Mula-App-main/src/main/kotlin/com/lamulaapp/controller/dto/LoginDto.kt

```diff
@@ -0,0 +1,16 @@
+package com.lamulaapp.controller.dto
+
+import com.lamulaapp.domain.Users
+import java.util.UUID
+import java.time.LocalDateTime
+
+data class LoginDto(
+    val id: UUID? = null,
+    val idUser: Users? = null,

```

Comment:
En los Dtos debemos usar siempre otros objetos Dtos no entities.
Las clases en Kotlin/Java se nombran en singular.
Estas mapeando todo el UserDto no solo el idUserDto por lo tanto, el nombre de ese campo debería ser userDto.

```suggestion
    val userDto: UserDto? = null,

```